packages/lab/src/interactable-card/InteractableCardGroup.tsx

```diff
@@ -0,0 +1,206 @@
+import {
+  ComponentPropsWithoutRef,
+  forwardRef,
+  KeyboardEvent,
+  SyntheticEvent,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { clsx } from "clsx";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+
+import { makePrefixer, useControlled, useForkRef } from "@salt-ds/core";
+import {
+  InteractableCardGroupContext,
+  SelectionVariant,
+  InteractableCardValue,
+} from "./InteractableCardGroupContext";
+import interactableCardGroupCss from "./InteractableCardGroup.css";
+
+export interface InteractableCardGroupProps
+  extends Omit<ComponentPropsWithoutRef<"div">, "onChange"> {
+  /**
+   * The default value. Use when the component is not controlled. Should be an array when `selectionVariant` is "multiselect".
+   */
+  defaultValue?: InteractableCardValue;
+  /**
+   * If `true`, the Interactable Card Group will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * The value. Use when the component is controlled.
+   */
+  value?: InteractableCardValue;
+  /**
+   * The selection variant.
+   */
+  selectionVariant?: SelectionVariant;
+  /**
+   * Callback fired when the selection changes.
+   * @param event
+   */
+  onChange?: (
+    event: SyntheticEvent<HTMLDivElement>,
+    value: InteractableCardValue
+  ) => void;
+}
+
+const withBaseName = makePrefixer("saltInteractableCardGroup");
+
+export const InteractableCardGroup = forwardRef<
+  HTMLDivElement,
+  InteractableCardGroupProps
+>(function InteractableCardGroup(props, ref) {
+  const {
+    children,
+    className,
+    value: valueProp,
+    defaultValue,
+    disabled,
+    onChange,
+    onKeyDown,
+    selectionVariant = "single",
+    ...rest
+  } = props;
+
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-interactable-card-group",
+    css: interactableCardGroupCss,
+    window: targetWindow,
+  });
+
+  const groupRef = useRef<HTMLDivElement>(null);
+  const handleRef = useForkRef(ref, groupRef);
+
+  const [value, setValue] = useControlled({
+    default: defaultValue,
+    controlled: valueProp,
+    name: "InteractableCardGroup",
+    state: "value",
+  });
+
+  const [elements, setElements] = useState<HTMLElement[]>([]);
+
+  useEffect(() => {
+    const childElements: HTMLElement[] = Array.from(
+      groupRef.current?.querySelectorAll(
+        ".saltInteractableCard:not([disabled])"
+      ) ?? []
+    );
+    setElements(childElements);
+  }, [children]);
+
+  const select = useCallback(
```

Comment:
in the group multiselect, clicking twice seem to unselect a card, but key nav doesn't (it might still be in the tbd but thought of mentioning it still just in case)

Reply:
where do you see this? In storybook the multiselect unselects with space, is it the focus ring making it less visible?

Reply:
ah, i see what is happening, so keyboard selection happens with spacebar but not with enter, that confused me, but if that's the intentional behaviour its fine

Reply:
yeah I think it should behave like a checkbox, so only space not enter